lang/test/org/partiql/lang/eval/evaluatortestframework/PartiQLCompilerPipelineFactory.kt

```diff
@@ -71,7 +71,7 @@ internal class PartiQLCompilerPipelineFactory : PipelineFactory {
         )
 
         val pipeline = PartiQLCompilerPipeline(
-            parser = SqlParser(ION, legacyPipeline.customDataTypes),
+            parser = PartiQLParserBuilder().ionSystem(ION).customTypes(legacyPipeline.customDataTypes).build(),
             planner = PartiQLPlannerBuilder.standard()
                 .options(plannerOptions)
                 .globalVariableResolver(globalVariableResolver)
```

Comment:
nit: we can replace this with `PartiQLCompilerPipeline.build { ... }` now, don't block the PR on this though